packages/flutter/lib/src/logging/logging.dart

```diff
@@ -0,0 +1,99 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:collection';
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/logging.dart';
+
+typedef void _RegisterServiceExtensionCallback(
+    {@required String name, @required ServiceExtensionCallback callback});
```

Comment:
I believe we want spaces after and before the braces, but the style guide would know for sure.

Reply:
kk.  Thanks!

Reply:
just use trailing commas and then dartfmt does the right thing (usually)